events/dvg_lotharingia.txt

```diff
@@ -0,0 +1,1204 @@
+			value = 0.1
+		}
+		
+		activate_law = law_type:law_autocracy
+		activate_law = law_type:law_monarchy
+		activate_law = law_type:law_hereditary_bureaucrats
+		
+		ruler = { remove_trait = immortal }
+		
+		c:LOT = { trigger_event = { id = dvg_lotharingia.5 popup = yes } }
+
+		ai_chance = {
+			base = 40
+		}
+	}
+	option = {
+		name = dvg_lotharingia.2.b
+		
+		
+		
+		add_radicals = {
+			value = -0.2
+		}
+		activate_law = law_type:law_autocracy
+		activate_law = law_type:law_appointed_bureaucrats
+		ruler = { kill_character = yes }
+		create_character = {
+			noble = yes
+			ruler = yes
+			first_name = "Jean"
+			last_name = "Bouillon"
+			culture = cu:burgundian
+			female = no
+			commander_rank = default
+			interest_group = ig_landowners
+			ig_leader = yes
+			ideology = ideology_traditionalist
+			birth_date = 1800.6.20
+			traits = {
+				
+				romantic
+				
+				pious
+				
+			}
+			# prestige = 0.1
+		}
+
+		ai_chance = {
+			base = 20
+		}
+	}
+	option = {
+		name = dvg_lotharingia.2.c
+		
+		
+		
+		add_radicals = {
+			value = 0.5
+		}
+		
+		c:LOT = { trigger_event = { id = dvg_lotharingia.5 popup = yes } }
+
+		ai_chance = {
+			base = 30
+		}
+	}
+   option = {
+		name = dvg_lotharingia.2.e
+		default_option = yes
+		add_modifier = {
+			name = modifier_army_purged
+			months = normal_modifier_time
+		}
+		ruler = { kill_character = yes }
+		
+		
+		add_radicals = {
+			value = -0.5
+		}
+		activate_law = law_type:law_migration_controls
+		activate_law = law_type:law_census_voting
+		ig:ig_landowners = {
+			remove_ideology = ideology_caudillismo
+			}
+                ig:ig_armed_forces = {
+			remove_ideology = ideology_caudillismo
+			}
+		create_character = {
+			noble = yes
+			first_name = "Jean"
+			last_name = "Bouillon"
+			culture = cu:burgundian
+			female = no
+			commander_rank = default
+			interest_group = ig_landowners
+			ig_leader = yes
+			ideology = ideology_traditionalist
+			birth_date = 1800.6.20
+			traits = {
+				
+				romantic
+				
+				pious
+				
+			}
+			
+		}
+		call_election = {
+				months = 1
+			}
+		ai_chance = {
+			base = 10
+		}
+	}
+}
+
+dvg_lotharingia.3 = {
+	type = country_event
+	placement = ROOT
+
+	title = dvg_lotharingia.3.t
+	desc = dvg_lotharingia.3.d
+	flavor = dvg_lotharingia.3.f
+	
+	event_image = {
+		video = "gfx/event_pictures/unspecific_sick_in_hospital.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/middleeast/courtroom_upheaval"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	trigger = { root.average_sol >= 12
+	}
+	
+	
+	# Liberte!
+	option = {
+		name = dvg_lotharingia.3.a
+		default_option = yes
+		
+		add_loyalists = {
+			value = .5
+		}
+		change_variable = {
+				name = aigle_var
+				add = -10
+			}
+		
+		
+		
+
+		ai_chance = {
+			base = 50
+		}
+	}
+	option = {
+		name = dvg_lotharingia.3.b
+		
+		
+		add_radicals = {
+			value = -.5

```

Comment:
-0.5. but does it actually work to give negative radicals?

Reply:
yes